content/en/community/roadmap.md

```diff
@@ -6,7 +6,7 @@ type: docs
 weight: 100
 ---
 
-The Jenkins X roadmap will be updated approximately once a month. Please join the office hours and the community Slack channels to ask questions and drive input on the future of the project!
+The Jenkins X roadmap will be updated approximately once a month. Please join our [office hours](https://jenkins-x.io/community/#office-hours) and [Slack channels](https://jenkins-x.io/community/#slack) to ask questions and drive input on the future of the project!
```

Comment:
One sentence per line :)

Reply:
> One sentence per line :)

That would be nice. It might take some time to get there and we might need to have some patience. 

I have been considering doing a big overhaul and migrating all Markdown files in one go. It is actually quite fast to do this. It would be easier to "enforce" this style once the bulk of the docs adheres to it.

@MarckK - do you want to change this, do you want me to change it (I can push an update) or do you want to merge?

Reply:
@hferentschik this might help in the future #2912

Reply:
>  this might help in the future #2912

yeah, already upvoted this one ;-) Not sure whether it supports checking for line per sentence. But I would be already keen to get all the other things it checks.

Reply:
They have discussions around it, so may be soon ;-) https://github.com/DavidAnson/markdownlint/issues/66

Reply:
It's a good rule. I've done the PR as a patch, so it's a fiddle to update. I'll just close this and do another.